mix.exs

```diff
@@ -73,4 +73,27 @@ defmodule VintageNetQMI.MixProject do
       links: %{"GitHub" => @source_url}
     ]
   end
+
+  # utility function that strips the original csv
+  # database file down to the fields we need. 
+  @doc false
+  def build_csv(path) do
+    csv_data =
+      path
+      |> File.read!()
+      |> String.trim()
+      |> String.split("\r\n")
+      |> Enum.map(&String.split(&1, ";"))
+      |> tl()
+      |> Enum.reject(fn [_, _, _, _, _, _, _, pro, _, _ | _] ->
+        String.length(pro) == 0
+      end)
+      |> Enum.map(fn [mcc, mnc, _, _, _, _, _, pro, _, _ | _] -> [mcc, mnc, pro] end)
+      |> Enum.uniq_by(fn [mcc, mnc, _] -> {mcc, mnc} end)
+      |> Enum.map(&Enum.join(&1, ";"))
+      |> Enum.join("\n")
+
+    _ = File.mkdir_p!("priv")
+    :ok = File.write!("priv/mcc-mnc.csv", csv_data)
```

Comment:
Does this create a directory under `_build`? It seems like the current directory should be the main directory. If it is, could you move it to `_build` so that there aren't any build products in the main source place. I don't know if you remember all of the problems we used to have with source `priv` vs. build `priv` with Nerves. Since the processing doesn't depend on env or target, I don't see how we could have the same issues, but it will bothers me to see things output to the source priv.

Reply:
oh yeah, easy update. on it